proto/data.proto

```diff
@@ -252,6 +252,22 @@ message Transaction {
   optional NodeList certain_nodes = 12 [(gogoproto.nullable) = false];
 }
 
+// Lease contains information about leader leases including the
+// expiration and lease holder.
+message Lease {
+  // The expiration is in unix nanos and is set according to the wall
+  // clock at the lease requestor / grantee.
+  optional int64 expiration = 1 [(gogoproto.nullable) = false];
+  // The leadership term for this lease.
+  optional uint64 term = 2 [(gogoproto.nullable) = false];
+  // The node ID of replica holding the lease.
```

Comment:
you're right. But we should be able to drop the NodeID, as the request is routed only to the ranges that belong to the group, and then all you need to know is that you're in the right store. I think NodeID especially can be confusing, especially with MultiRaft.NodeID around.